fs/_ftp_parse.py

```diff
@@ -29,7 +29,7 @@
     \s+?
     (\d+)
     \s+?
-    (\w{3}\s\d{2}\s+[\w:]+)
+    (\w{3}\s+\d+\s+[\w:]+)
```

Comment:
So this is the day number? Maybe the regex should be tighter and allow 1 or 2 digits?